pkg/envoy/proxy.go

```diff
@@ -134,12 +134,12 @@ func (e *envoy) args(fname string, bootstrapConfig string) []string {
 	if e.ProxyConfig.LogAsJSON {
 		startupArgs = append(startupArgs,
 			"--log-format",
-			`{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"envoy %n","msg":"%j"}`,
+			`{"level":"%l","time":"%Y-%m-%dT%T.%fZ","scope":"envoy %n","msg":"%j","loc":"%g:%#","tid":"%t"}`,
 		)
 	} else {
 		// format is like `2020-04-07T16:52:30.471425Z     info    envoy config   ...message..
 		// this matches Istio log format
-		startupArgs = append(startupArgs, "--log-format", "%Y-%m-%dT%T.%fZ\t%l\tenvoy %n\t%v")
+		startupArgs = append(startupArgs, "--log-format", "%Y-%m-%dT%T.%fZ\t%l\tenvoy %n:%t\t[%g:%#] %v")
```

Comment:
I don't think this matches the logging format of Istio.


example:
`2022-10-18T17:47:47.486137Z     info    some/file.go:872     some log        thread=1234`

is what this would look like 

emitted with

```go
log.WithLabels("thread", 1234).Infof("some log")
```

and --log_caller=all

Reply:
Updated.